andela/eagle-bn-frontend
# Ft verify user #169889344

#### What does this PR do?
Verify user email
#### Description of Task to be completed?
- Create verify email page 
- write tests 
- write story 
#### How should this be manually tested?
- You can test the component states by running storybook :
`npm run storybook`
- You can also run npm run dev:open
`go to users/verify/:token`
#### Any background context you want to provide?
N/A
#### What are the relevant pivotal tracker stories?
https://www.pivotaltracker.com/story/show/169889344
#### Screenshots (if appropriate)
<img width="870" alt="Screen Shot 2019-12-04 at 10 19 58" src="https://user-images.githubusercontent.com/49560334/70125700-fe389480-167f-11ea-9ec7-e41848f6d30d.png">
<img width="1222" alt="Screen Shot 2019-12-04 at 10 19 40" src="https://user-images.githubusercontent.com/49560334/70125775-3049f680-1680-11ea-983a-01156ad8f8d7.png">
<img width="923" alt="Screen Shot 2019-12-04 at 10 19 52" src="https://user-images.githubusercontent.com/49560334/70125785-33dd7d80-1680-11ea-8c1c-31b40e9ace23.png">

#### Questions:
N/A
